framework/core/render_pass.cpp

```diff
@@ -352,6 +401,11 @@ void RenderPass::create_renderpass(const std::vector<Attachment> &attachments, c
 		}
 	}
 
+	if (attachments.size() != load_store_infos.size())
```

Comment:
If this is required, it should be moved to the top of the function, or maybe into the `RenderPass` constructor, where this function is called.

Reply:
Done